src/overlay/OverlayManagerImpl.cpp

```diff
@@ -253,7 +256,11 @@ OverlayManagerImpl::start()
     {
         mTimer.async_wait(
             [this]() {
-                storeConfigPeers();
+                storeConfigPeers(true);
+                mPeerIPTimer.expires_from_now(PEER_IP_RESOLVE_DELAY);
```

Comment:
I wonder if we should just call this code from `tick` instead of using a dedicated timer? Less things to keep track of

Reply:
Alternatively, you could just call
```
storeConfigPeers(true);
storeConfigPeers(false);
```
to avoid duplicating the scheduling.

Reply:
I ended up going with your latter suggestion. With the former I think we'd end up resolving peers too often (although we can check timestamps in tick to resolve only after some time, but that would use a timer anyway)